R/z-packageStartupMessage.R

```diff
@@ -0,0 +1,6 @@
+.onAttach <- function(libname, pkgname) {
+  packageStartupMessage(
+    "We'd like to hear your feedback and thoughts! Let us know by ",
```

Comment:
This is a good opportunity to use `{cli}` to control how often it's displayed. I wanted to comment on this a while back but forgot.